razorpay/client.py

```diff
@@ -80,6 +83,26 @@ def _get_version(self):
             pass
         return version
 
+    def _get_app_details_ua(self):
+        appDetailsUa = ""
+
+        app_details = self._get_app_details()
+
+        for app in app_details:
+            if 'title' in app and isinstance(app['title'], str):
+                appUa = app['title']
+                if 'version' in app and not isinstance(app['version'], list):
+                    appUa += "{}{}".format('/', app['version'])
```

Comment:
appUa += "{}{}".format('/', app['version'])
to 
app_ua += "/{}".format(app['version'])